server/server.js

```diff
@@ -59,13 +59,77 @@ app.get('/recipes', (req, res) => {
   utils.getRecipes(req.query, (error, results) => {
     if (error) {
       res.send(error);
-    } 
+    }
     console.log('body: ', JSON.parse(results));
     let parsed = JSON.parse(results);
     res.json(parsed.matches);
   });
 });
 
+<<<<<<< HEAD
+=======
```

Comment:
May want to check this, as the <<<< HEAD is showing up which usually indicates there is a fix that needs to be made.

Reply:
Thanks Tristan, I have updated.